### PR TITLE
Google drive uploader

### PR DESCRIPTION
**Pull request by DenisUA**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,13 @@ By default ReqRes will use `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_RE
 
 `REQRES_UPLOAD=1 REQRES_RSPEC=1 bundle exec rspec --order=defined`
 
+## Upload to Goorle Drive
```
Comment (rilian) [inline README.md]: typo

File: README.md [hunk 2]
```diff
@@ -46,6 +46,13 @@ By default ReqRes will use `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_RE
...
+
+First, follow “Create a client ID and client secret” in [this page](https://developers.google.com/drive/web/auth/web-server) to get a client ID and client secret for OAut. Update `GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET` in your .env file.
```
Comment (rilian) [inline README.md]: use straight quotes please

File: lib/reqres_rspec/uploaders/google_drive.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
+require 'google/api_client'
+require 'google_drive'
+
+module ReqresRspec
+  module Uploaders
+    class GoogleDrive
+      def initialize
```
Comment (rilian) [inline lib/reqres_rspec/uploaders/google_drive.rb]: plz add a comment with link from where you took this snipper

File: README.md (hunk 2 above)
Comment (sudodoki) [inline README.md]: `OAut` -> `Owut`, maybe?

File: README.md [hunk 4]
```diff
@@ -44,7 +44,27 @@ Documentation will be put into your application's `/doc` folder
...
+
+`REQRES_UPLOAD=1 REQRES_RSPEC=AmazonS3 bundle exec rspec --order=defined`
+
+Also you can run
+
+`REQRES_UPLOAD=1 REQRES_RSPEC=AmazonS3 AWS_ACCESS_KEY_ID='--your access key id--' AWS_SECRET_ACCESS_KEY='--your secret access key--' AWS_REQRES_BUCKET='--your bucket name--'  bundle exec rspec --order=defined`
+
+## Upload to Goorle Drive
+
+First, follow “Create a client ID and client secret” in [this page](https://developers.google.com/drive/web/auth/web-server) to get OAuth credentials. Update `GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET` in your .env file.
```
Comment (rilian) [inline README.md]: Not everyone who use the gem, uses `.env`, or wants to put these credentials into the file. Please prioritize example of a 1-liner command to run in the shell. Thanks

**Comment by DenisUA** [comment]
Please review

**Comment by rilian** [comment]
checking

File: README.md [hunk 5]
```diff
@@ -108,8 +132,8 @@ describe 'Something', reqres_section: 'Foo' do
 end
 ```
 
-In this case all the `reqres_sections` can be used for grouping colleced data into section, and `reqres_title` will become human readable titles:
-[![Cusomized titles](http://i57.tinypic.com/2581lw9.jpg)](http://i57.tinypic.com/2581lw9.jpg)
+In this case all the `reqres_sections` can be used for grouping collected data into section, and `reqres_title` will become human readable titles:
+[![Customized titles](http://i57.tinypic.com/2581lw9.jpg)](http://i57.tinypic.com/2581lw9.jpg)
```
Comment (sudodoki) [inline README.md]: Nice catch :+1:

**Comment by john-denisov** [comment]
ok

File: README.md [hunk 6]
```diff
@@ -40,11 +40,35 @@ by default `reqres_rspec` is not active (this may be configured!). To activate i
...
-`REQRES_UPLOAD=1 REQRES_RSPEC=1 bundle exec rspec --order=defined`
+```
+AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY
+AWS_REQRES_BUCKET
+```
+
+Then run
+
+`REQRES_UPLOAD=1 REQRES_RSPEC=AmazonS3 bundle exec rspec --order=defined`
```
Comment (rilian) [inline README.md]: this should be

`REQRES_RSPEC=1 REQRES_UPLOAD=AmazonS3` ...

**Comment by DenisUA** [comment]
pls review

**Comment by rilian** [comment]
:+1:

File: README.md (hunk 6 above)
Comment (creeonix) [inline README.md]: I'd say that this should be configured in ReqRrs.configure - anyway uploaded configuration will require setup :)

Comment (rilian) [inline README.md]: yes, i plan to rewrite readme anyway. it's not very organised and it must be split to smaller files

**Pull request merged**